### DISCUSSION TITLE
No comment field for Locations?

### DISCUSSION THREAD

**Discussion by mlindgren80** [netbox-community/netbox#11114] [Ideas]
We use Locations for our network closets in buildings, and it would be nice to have a comment field which would let us give directions for the network closet in case it is difficult to find.  Is this possible, or am I using Locations incorrectly?

**Comment by mtinberg** [comment]
Locations don't currently have a long-form comment field, and you could always file a feature request to have one added, but in the interim you can make a custom field for Location to put access information into


—

Mark Tinberg ***@***.***>

Division of Information Technology-Network Services

University of Wisconsin-Madison


________________________________
From: mlindgren80 ***@***.***>
Sent: Monday, December 5, 2022 3:32 PM
To: netbox-community/netbox ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [netbox-community/netbox] No comment field for Locations? (Discussion #11114)


We use Locations for our network closets in buildings, and it would be nice to have a comment field which would let us give directions for the network closet in case it is difficult to find. Is this possible, or am I using Locations incorrectly?
